manifests/00-crd-operator-config.yaml

```diff
@@ -0,0 +1,23 @@
+# operator.openshift.io Console is the private configuration for the
+# console operator.
+#
+# config.openshift.io Console is the public (user facing) configuration
+# for the console.  See this CRD
```

Comment:
Lets swap these two comments and if https://github.com/openshift/cluster-config-operator/pull/11 got merged lets add link for the CRD :)

Reply:
This is confusing enough, I'll delete the comment here, but consider putting something in README or somewhere as a follow-up.  It should be clear how these work.